docs/modules/Option.ts.md

```diff
@@ -63,15 +63,7 @@ Added in v2.0.0
   - [isOption](#isoption)
   - [isSome](#issome)
 - [lifting](#lifting)
-  - [lift2](#lift2)
```

Comment:
Was removing lift2 intentional?

Reply:
Going to add it back, to keep an alternative api for users to migrate to if required.